packages/terra-collapsible-menu-view/src/CollapsibleMenuView.module.scss

```diff
@@ -55,6 +59,11 @@
     &:first-child {
       padding-left: 0;
     }
+
+    :focus {
+      box-shadow: var(--terra-collapsible-menu-view-item-focus-box-shadow, none);
+      outline: var(--terra-collapsible-menu-view-item-focus-outline, 2px dashed #000);
+    }
```

Comment:
The one for the menu button and the face up item right? I noticed that for the menu button the top outline was getting cut off because it was too big, so I just reduced the outline size to make things look nicer